core-web/package.json

```diff
@@ -87,10 +87,10 @@
     "@tiptap/starter-kit": "^2.0.0-beta.218",
     "@tiptap/suggestion": "^2.0.0-beta.218",
     "@webcomponents/webcomponentsjs": "^2.6.0",
-    "chart.js": "^3.3.2",
+    "chart.js": "^4.2.1",
```

Comment:
Did you check dotCDN?

Reply:
I checked the config, and it does not show any problem at the IDE, but I can't find where or how to run that chart.